geonotebook/kernel.py

```diff
@@ -404,8 +407,16 @@ def handle_comm_open(self, comm, msg):
 
         # THis should be handled in a callback that is fired off
         # When set protocol etc is complete.
-        self.geonotebook.add_layer(None, name="osm_base", layer_type="osm",
-                                   vis_url="http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png")
+        self.geonotebook.add_layer(
+            None, name="osm_base", layer_type="osm",
+            vis_url="http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+            system_layer=True)
+
+        self.geonotebook.add_layer(
```

Comment:
The annotation layer exists to create the potential for annotations.  Even if there are no annotations there should always be the potential for annotations.